src/grammar-parser/grammar-parser.jison

```diff
@@ -15,7 +18,7 @@
 [A-Za-z\.]+(?=[(])                                                                              {return 'FUNCTION';}
 [A-Za-z]{1,}[A-Za-z_0-9]+                                                                       {return 'VARIABLE';}
 [A-Za-z_]+                                                                                      {return 'VARIABLE';}
-[0-9]+                                                                                          {return 'NUMBER';}
```

Comment:
This regular expression was replaced with one that supports unary minus/plus, and decimal points, and we just let javascript's parseFloat deal with that instead of describing that in the lex unnecessarily